input/pagecontent/index.md

```diff
@@ -55,6 +55,7 @@ This guide is divided into several pages which are listed at the top of each pag
 - [FHIR Artefacts](artifacts.html): These pages provide detailed descriptions and formal definitions for all the FHIR artefacts defined in this guide.
   - [Profiles and Extensions](profiles-and-extensions.html): This page describes the profiles and extensions that are defined in this guide to support electronic requesting. Each profile page includes a narrative description and guidance, formal definition and a "Notes" section which summarises the supported search transactions for each profile. Although the guidance typically focuses on the profiled elements, it may also may focus on un-profiled elements to aid with implementation.
   - [Terminology](terminology.html): This page lists the value sets and code systems supported in this guide.
+  - [Capability Statements](capability-statements.html): This page defines the expected FHIR capabilities of AU eRequesting clients and servers.
```

Comment:
This text is generically placeholder, i.e. 'clients and servers'. I can't see the value in being generically placeholder so that it doesn't match final or current state. 

Final state = listing all actors.
Current state = listing the actor where there is a capability statement defined

Please pick one of the above, likely current state is best.